dts/bindings/sensor/meas,htu21d.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021, Jonas Norling
+# SPDX-License-Identifier: Apache-2.0
+
+description: TE Connectivity HTU21D temperature and humidity sensor
+
+compatible: "meas,htu21d"
```

Comment:
`meas` is the vendor prefix for Measurement Specialties, but the vendor in the description is "TE Connectivity".

This looks like a violation of the rules: https://docs.zephyrproject.org/latest/guides/dts/bindings.html#vendor-prefixes

What's going on?

Reply:
Measurement Specialties was bought by TE Connectivity last decade. They seem to use both names, in addition to "MEAS", when marketing this range of sensors. meas in in the vendor file, both in Zephyr and Linux, whereas TE is not, so it felt appropriate to keep using that prefix.
Updated yaml file to include both names.